.github/workflows/build.yml

```
@@ -6,6 +6,7 @@ on:
     paths:
       - '27.2/**'
       - '28.2/**'
+      - '28.3/**'
```

```suggestion
      - '28.3rc1/**'
```

It all LGTM, but this is needed to kick off the CI.

---

Added.